pkgs/applications/misc/dmenu/default.nix

```diff
@@ -13,10 +13,17 @@ stdenv.mkDerivation rec {
 
   inherit patches;
 
-  postPatch = ''
-    sed -ri -e 's!\<(dmenu|dmenu_path|stest)\>!'"$out/bin"'/&!g' dmenu_run
-    sed -ri -e 's!\<stest\>!'"$out/bin"'/&!g' dmenu_path
-  '';
+  postPatch =
+    let
+      # Allow users to set the config.def.h file containing the configuration
+      configFile =
+        if lib.isDerivation conf || builtins.isPath conf
+        then conf else lib.writeText "config.def.h" conf;
```

Comment:
```suggestion
        if lib.isDerivation conf || builtins.isPath conf then 
          conf 
        else
          lib.writeText "config.def.h" conf;
```

Reply:
Why though? Formatter doesn't seem to care.

Reply:
OK.